src/main/kotlin/us/ihmc/ci/TagParser.kt

```diff
@@ -88,14 +89,10 @@ object TagParser
     */
    private fun assembleTestClasspath(testProject: Project, contextClasspathUrls: ArrayList<URL>, selectorPaths: HashSet<Path>)
    {
-      // TODO:
-      val java = testProject.convention.getPlugin(JavaPluginConvention::class.java)
-//      val java = testProject.convention.getPlugin(JavaLibraryPlugin::class.java)
-//      testProject.plugins.
-//      testProject.configurations.getByName("default").forEach { file ->
-      java.sourceSets.getByName("main").compileClasspath.forEach { file ->
-         addStuffToClasspath(file, contextClasspathUrls, selectorPaths)
-      }
+      val java = testProject.extensions.getByType<JavaPluginExtension>()
+//      java.sourceSets.getByName("main").compileClasspath.forEach { file ->
+//         addStuffToClasspath(file, contextClasspathUrls, selectorPaths)
+//      }
```

Comment:
I don't understand why this was declared, but it added lots more tests to the selectorPaths, which caused lots of tests that werent even being run to print that they were going to be run